docs/csi.md

```diff
@@ -3,13 +3,13 @@
 So let's take each part of the [CSIVolumeSource](https://github.com/kubernetes/api/blob/71efbb18d63cd30604981514ac623a6be1d413bb/core/v1/types.go#L1743-L1771):
 
 - for the `Driver` string field, it needs to be ["csi-driver-projected-resource.openshift.io"](https://github.com/openshift/csi-driver-shared-resource/blob/1fcc354faa31f624086265ea2228661a0fc2e7b1/pkg/client/client.go#L28).
-- for the `VolumeAttributes` map, this driver currently adds the "share" key (which maps the the `SharedResource` instance your `Pod` wants to use) in addition to the
+- for the `VolumeAttributes` map, this driver currently adds the "share" key (which maps the the `SharedConfigMap` OR `SharedSecret` instance your `Pod` wants to use) in addition to the
```

Comment:
```suggestion
- for the `VolumeAttributes` map, this driver currently adds the "share" key (which maps the `SharedConfigMap` OR `SharedSecret` instance your `Pod` wants to use) in addition to the
```

Reply:
yeah actually the key name is not correct as well

it is sharedConfigMap or sharedSecret

I'll make that changes as well here